src/main/java/org/jenkinsci/plugins/stashNotifier/NotifierSelectors/HttpNotifierSelector.java

```diff
@@ -1,4 +1,4 @@
-package org.jenkinsci.plugins.stashNotifier;
+package org.jenkinsci.plugins.stashNotifier.NotifierSelectors;
```

Comment:
I appreciate the effort to keep similar implementations together, but unfortunately this is a breaking change.

Can we leave it in the original package to minimize the work consumers will have to do?